src/Avalonia.X11/X11Platform.cs

```diff
@@ -384,8 +384,15 @@ public class X11PlatformOptions
         /// will likely brick GLib machinery since it's not aware of managed Exceptions
         /// This property allows to inspect such exceptions before they will be ignored
         /// </summary>
-        public Action<Exception>? ExterinalGLibMainLoopExceptionLogger { get; set; }
-        
+        public Action<Exception>? ExternalGLibMainLoopExceptionLogger { get; set; }
+
+        [ObsoleteAttribute("Use ExternalGLibMainLoopExceptionLogger instead")]
+        public Action<Exception>? ExterinalGLibMainLoopExceptionLogger
```

Comment:
We likely won't backport this, renaming the property directly for v12 sounds like an acceptable breaking change, especially for such a niche property.